Specs/spec-main.js

```diff
@@ -58,6 +58,8 @@
 
         /*global jasmineRequire,jasmine,exports,specs*/
 
+        Cesium.RequestScheduler.prioritize = false;
```

Comment:
Looks like an OK place to me, I don't think there is a `beforeAll` that applies to all suites.